apps/radix-docs/src/demos/primitives/collapsible/collapsible-demo.ts

```diff
@@ -19,9 +19,9 @@ import { LucideAngularModule, UnfoldVertical, X } from 'lucide-angular';
     ],
     template: `
         <div class="CollapsibleRoot" #collapsibleRoot="collapsibleRoot" [open]="true" rdxCollapsibleRoot>
-            <div style="display: flex; align-items: center; justify-content: space-between;">
+            <div style="display: flex; align-items: center; justify-content: space-between; gap: 1em;">
                 <span class="Text" style="color: white;">&#64;peduarte starred 3 repositories</span>
-                <button class="IconButton" rdxCollapsibleTrigger>
+                <button class="IconButton" style="flex-shrink: 0;" rdxCollapsibleTrigger>
```

Comment:
use [flex](https://tailwindcss.com/docs/flex-shrink) selectors, like `shrink-0` etc

Reply:
it seem to me, we don't use tailiwind in the component examples